Makefile

```diff
@@ -88,7 +88,7 @@ run-usdf-dev: export CM_DATABASE_URL=postgresql://cm-service@${CM_DATABASE_HOST}
 run-usdf-dev: export CM_ARQ_REDIS_URL=redis://${CM_ARQ_REDIS_HOST}:6379/1
 run-usdf-dev: export CM_DATABASE_PASSWORD=$(shell kubectl --cluster=usdf-cm-dev -n cm-service get secret/cm-pg-app -o jsonpath='{.data.password}' | base64 --decode)
 run-usdf-dev: export CM_ARQ_REDIS_PASSWORD=$(shell kubectl --cluster=usdf-cm-dev -n cm-service get secret/cm-redis-app -o jsonpath='{.data.password}' | base64 --decode)
-run-usdf-dev: export CM_DATABASE_ECHO=true
+run-usdf-dev: export CM_DATABASE_ECHO=True
```

Comment:
Why?  I verified lower-case "true" works.  This is make syntax, not Python...

Reply:
It didn't work for me at some point, somewhere it was complaining that it had a string in stead of a boolean.   Maybe pydantic version dependent?